src/main/java/org/fcrepo/client/FcrepoResponse.java

```diff
@@ -55,6 +72,36 @@ public FcrepoResponse(final URI url, final int statusCode,
     }
 
     /**
+     * {@inheritDoc}
+     * <p>
+     * Implementation note: Invoking this method will close the underlying {@code InputStream} containing the entity
+     * body of the HTTP response.
+     * </p>
+     *
+     * @throws IOException if there is an error closing the underlying HTTP response stream.
+     */
+    @Override
+    public void close() throws IOException {
```

Comment:
Yeah, why not. And I would get rid of `body` and just call into that underlying response directly.